registry.json

```diff
@@ -50,53 +50,57 @@
     "listed": true,
     "version": "1.1.0"
   },
+  "CLSS.Constants.CollectionPool": {
+    "listed": true,
+    "version": "1.0.0"
+  },
   "CLSS.Constants.DefaultRandom": {
     "listed": true,
-    "version": "1.2.0"
```

Comment:
This version should not be updated (As the others below). Once published, it should stay published unless there is a real problem having it around.

Reply:
(Because UnityNuGet will publish the upper versions automatically)

Reply:
PR updated to only contain new package(s).